doc/mcuboot/conf.py

```diff
@@ -101,9 +101,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '1.6.0-rc2'
+version = '1.6.0.99'
```

Comment:
@ru-fu Version 1.6.0 > x< {1.6.1, 1.7.0}   1.6.0.99 or 1.6.99?

Reply:
No idea. @carlescufi ?

Reply:
This should be `1.6.99`.